app/assets/javascripts/assignment.js

```diff
@@ -1,16 +1,22 @@
 $(document).ready(function() {
+    // enable bootstrap tooltips
+    $("[data-toggle='tooltip']").tooltip('enable');
     var copyTextBtn = $('.copy-text-btn');
-
     $(copyTextBtn).on('click', function() {
         var copyTextarea = $('.copy-text');
         copyTextarea.focus();
         copyTextarea.select();
         try {
             var successful = document.execCommand('copy');
-            var msg = successful ? 'successful' : 'unsuccessful';
-            console.log('Copying text command was ' + msg);
+            console.log('Copied successfully');
         } catch (err) {
             console.log('Unable to copy');
         }
+        if(successful === true) {
```

Comment:
Remember that boolean values are already the bread and butter of `if` statements. `successful === true` will evaluate to a boolean, but so will `successful` by itself.

Reply:
Right, can't believe I didn't realize what I did! Thanks!

Reply:
I was just too excited about threequal.

Reply:
I mean who wouldn't be